### PR TITLE
Run engine unit-tests on Windows in the presubmits.

### PR DESCRIPTION
**Pull request by chinmaygarde**
Also removes a redundant Windows build and the old run_tests.sh script.

### CODE REVIEW

File: .cirrus.yml [hunk 1]
```diff
@@ -27,12 +27,12 @@ task:
...
 
   matrix:
-    - name: build_and_test_host
+    - name: build_and_test_host_debug
       compile_host_script: |
         cd $ENGINE_PATH/src
         ./flutter/tools/gn --unoptimized --full-dart-sdk
         ninja -C out/host_debug_unopt
-      test_host_script: cd $ENGINE_PATH/src && ./flutter/testing/run_tests.sh host_debug_unopt
+      test_host_script: $ENGINE_PATH/src/flutter/testing/run_tests.py --variant host_debug_unopt
```
Comment (dnfield) [inline .cirrus.yml]: This LG but maybe we can document how to run the tests somewhere on the wiki, or update the doc if it's already there.

**Comment by chinmaygarde** [comment]
Hmm. I don't know why the flow_unittests failed on the bots when they pass locally.

**Comment by liyuqian** [comment]
Windows could be tricky... Feel free to go to https://console.cloud.google.com, create a Windows VM with the exact image used by Cirrus, and reproduce it there.

**Comment by cbracken** [comment]
@chinmaygarde @liyuqian any plans to continue with this PR?

**Comment by chinmaygarde** [comment]
No. These tests are enabled on LUCI already I'd rather not add to the Cirrus pile ATM.

**Pull request closed without merging**